modules/local/align/bwamem2_pysam.nf

```diff
@@ -18,7 +18,7 @@ process ALIGN_BWAMEM2 {
     script:
     def prefix = task.ext.prefix ?: "${meta.id}"
     def cov = '0.1'
-    if (sp_richness > 300) {
+    if (sp_richness > 150) {
         cov = '0.01'
```

Comment:
I missed this one.. would it make sense to have these as parameters (with defaults). In case a particular sample needs some tweaking for example?

Reply:
Mmmmm. So the user can control here? Yes, we could make this a parameter. Defaults will be based on our observations during the tool optimisation. I'll work on it